pypeit/datamodel.py

```diff
@@ -476,6 +476,11 @@ def _validate(self):
 from pypeit import masterframe
 from pypeit import msgs
 
+# TODO: We need methods that:
+#   - construct an rst table that documents the datamodel.
```

Comment:
There are methods that do this for specific DataModels, e.g. `SpecObj`
via `build_specobj_rst.py`.   I suggest we add it to this TODO

Reply:
I expanded this to read:
```
# TODO: There are methods in, e.g., doc/scripts/build_specobj_rst.py that output
# datamodels for specific datacontainers.  It would be useful if we had
# generalized methods in this base class that
#   - construct an rst table that documents the datamodel.
#   - construct an rst table that documents each Table component of the datamodel
#     (like a nested DataContainer)
```
Is that what you mean?

Reply:
yes, thanks